### PR TITLE
Embed

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
I haven't been able to get this to work 100%

### CODE REVIEW

File: inst/templates/socialMedia.mustache [hunk 1]
```diff
@@ -1,17 +1,24 @@
 <div id="socialHolder">
-  <a href="#" target="_blank">
+  <a href="{{githubLink}}" target="_blank">
```
Comment (ldecicco-USGS) [inline inst/templates/socialMedia.mustache]: @mwernimont any idea why this doesn't seem to do anything? When I build with this PR, I still get in source:
image

(the blank href being the annoying part)

Comment (mwernimont) [reply inst/templates/socialMedia.mustache]: Does it need 3 brackets? {{{}}} I'm just spit balling here, but will look at the rest of the code quick

**Comment by mwernimont** [comment]
@ldecicco-USGS what does the yaml look like for {{githublink}} etc... We may just be trying attack this wrong.

**Comment by ldecicco-USGS** [comment]
I think it's finally working!

image


Exception is some style, but I think now smarter design/css eyes can take it from here.

File: inst/resource.library.yaml [hunk 2]
```diff
@@ -58,6 +58,7 @@
     facebook-logo: lib-facebook-logo
     embed-logo: lib-embed-logo
     css: lib-social-media-css
+    social-links:
```
Comment (aappling-usgs) [inline inst/resource.library.yaml]: does this need a resource after the colon?

Comment (ldecicco-USGS) [reply inst/resource.library.yaml]: in fact, leaving it blank I think is important

**Pull request merged**